docs/guides/securing-platformatic-db.md

```diff
@@ -0,0 +1,42 @@
+# Securing Platformatic DB with Authorization
+
+## Introduction
+
+Authorization in Platformatic DB is **role-based**. User authentication and the
+assignment of roles must be handled by an external authentication service.
+Take a look to at the reference documentation for [Authorization](/docs/reference/db/authorization).
+
+The goal of this simple guide is to protect an API built with Platformatic DB
+with the use of a shared secret, that we call `adminSecret`. We want to prevent
+any user that is not an admin to access the data.
+
+:::info
+The use of an `adminSecret` is a simplistic way of securing a system.
+It is a crude way for limiting access and not suitable for production systems.
```

Comment:
Is there also an alternative [to this](https://github.com/platformatic/platformatic/pull/656/files#r1096531840) that can be used in production?

Reply:
I don't understand the question.

Reply:
Here you say that:
> The use of an `adminSecret` is a simplistic way of securing a system [...] not suitable for production systems

So, I'm wondering if there is a more "production-ready" way to obtain the same result of blocking all of the entities instead of manually selecting each one of them as mentioned by you and @marcopiraccini [here](https://github.com/platformatic/platformatic/pull/656/files#r1096544088)

Reply:
What's not production ready is the usage of an adminSecret. It's insecure as it will be shared by many consumers. You can use `JWT` or Webhooks to validate the request.

The comment has nothing to do with the "roles" definition, which will stay the same.